#test exception 8

int main() {
  return 1 && 0 || 1 / (1 && 0 || 2 / !2);
}
